EFS filesystem support (#19)
pvcy/enterprise-install: aws/terraform/modules/elastic-file-system/README.md

# Elastic File System Module

> This module creates an Elastic Files System (EFS) in AWS, as well as necessary network resources to allow access to the file system from your cluster. It also provisions an EFS-backed Kubernetes StorageClass. This is used by `privacy-api` from Privacy Dynamics, which requires persistent storage that can be used simultaneously across multiple pods and nodes.

Reviewer: Typo (`File(s)`).
Let's restate the purpose a bit to mention that the data is temporary and avoid any specific pod name mentions (these could change in the future). 

```suggestion
This module creates a file system in [Amazon Elastic File System (EFS)](https://docs.aws.amazon.com/efs/latest/ug/whatisefs.html), as well as necessary network resources to allow access from your cluster. It also provisions an EFS-backed Kubernetes StorageClass. This is used by the Privacy Dynamics application to store temporary data that can be used simultaneously from multiple pods and nodes.
```

## Resources

The Terraform files provided here will create the following resources:

- A Security Group connected to your cluster's existing, primary VPC (any secondary VPCs are not considered)
- An Ingress Rule attached to the Security Group, allowing TCP traffic on port 2049

> - A Egress Rule attached to the Security Group, allowing all outbound traffic

Reviewer 2: ```suggestion
- An Egress Rule attached to the Security Group, allowing all outbound traffic
```

> - An Elastic File System

Reviewer: Amazon Elastic File System is a service that uses "file systems". I think "EFS file system" is the most clear, even though it seems completely redundant :sweat_smile: 

```suggestion
- An EFS file system
```
